crates/l2/proposer/l1_committer.rs

```diff
@@ -28,7 +29,7 @@ use tracing::{error, info};
 
 use super::errors::BlobEstimationError;
 
-const COMMIT_FUNCTION_SELECTOR: [u8; 4] = [132, 97, 12, 179];
+const COMMIT_FUNCTION_SIGNATURE: &'static str = "commit(uint256,bytes32,bytes32,bytes32)";
```

Comment:
Clippy says the `'static` is redundant:
```suggestion
const COMMIT_FUNCTION_SIGNATURE: &str = "commit(uint256,bytes32,bytes32,bytes32)";
```